gui/src/app/FileEditor/TextEditor.tsx

```diff
@@ -104,16 +104,17 @@ const TextEditor: FunctionComponent<Props> = ({
 
   useEffect(() => {
     if (!editorInstance) return;
-    if (!hintTextOnEmpty) return;
+    if (!contentOnEmpty) return;
     if (text || editedText) {
       return;
     }
-    const contentWidget = createHintTextContentWidget(hintTextOnEmpty);
+    console.log("--- contentOnEmpty:", contentOnEmpty);
```

Comment:
Do we want to leave this in for production, or is it a debug check?

Reply:
oops. removed.